app/src/test/java/org/gnucash/android/test/unit/export/QifExporterTest.java

```diff
@@ -144,4 +149,46 @@ public void multiCurrencyTransactions_shouldResultInMultipleQifFiles(){
         assertThat(file.length()).isGreaterThan(0L);
     }
 
+    //@Test
+    public void description_and_memo_field_test() {
```

Comment:
Alright, thanks for the suggestion. I do, however, disagree that camel-case is an appropriate syntax for long test names. It is completely unreadable for anything longer than three words.
Even the name of the existing test combines the two standards, which is even worse, in my opinion.